### DISCUSSION TITLE
Access to MLPerf Traces

### DISCUSSION THREAD

**Discussion by jw12yoon** [accel-sim/accel-sim-framework#194] [General]
Hi all,

I have a question regarding access to mlperf traces for volta gpu simulation in SASS mode.
In the released config directory file : define-standard-cfgs.yml, there are commands referring to mlperf training/inference simulations.
However, when using the provided get-accel-sim-traces.py to collect the SASS traces, it seems mlperf traces are not available on the purdue univ. ftp server. 
Is there a way to get access to the mlperf traces? (volta/ampere GPU)
Or are mlperf traces not available yet to the public, due to its huge size?

Thanks.

**Comment by JRPan** [comment]
We have not publish the traces. Those in the config files were added for Principle Kernel Analysis integration. It takes forever to simulate MLPerf without PKA, and yes it's way too big. Are you trying to use PKA? If yes I can forward your request to the authors.

**Reply by jw12yoon** [reply]
My original intention was to check the availability of MLPerf traces, but as you mentioned, it seems like it is impossible to run MLPerf without PKA. Yes, if you could forward my request to the authors of PKA, it would be a great help. I will try to use PKA for simulation of MLPerf workloads. Thank you for your help!
